Payment.java

```diff
@@ -0,0 +1,68 @@
+import java.util.Scanner;
+
+public class Payment {
+    String Card;
+    int cash;
+    static int cashChange = 0;
+
+
+    int showPayment() {
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("[결제 방식 선택]");
+        System.out.println("[1] 현금");
+        System.out.println("[2] 카드 (부가세 10% 적용)");
+        System.out.println();
+        System.out.print("사용자 입력 > ");
+
+        int select_Payment_Method = sc.nextInt();
```

Comment:
자바 컨벤션을 따르면
```suggestion
int paymentMethod = scanner.nextInt();
```
처럼 될 것 같아요!